README.md

```diff
@@ -117,6 +119,7 @@ workflow:
 ```
 If you are processing a small number of files (less than 10) you can set this option to `False`.
 Or you can also set the kerberos authentication via the Snakemake rules.
+A Kerberos keytab file needs to be generated and uploaded to the REANA client to succesfully run this example.
```

Comment:
Hmm, I wonder whether we shouldn't make Kerberos optional, since it also applies at CERN only. Perhaps we could have two configuration files, `reana.yaml` and `reana-cern.yaml`, and plug the Kerberos and the CERN Harbor pull-through cache only for the second.  This may increase complexity a bit, but the example would be runnable also outside of the CERN context.